cloud/wfm/src/index.ts

```diff
@@ -15,3 +15,6 @@ export {BaseTask} from './task/BaseTask';
 
 // Repositories
 export {ExecutorRepository} from './executor/ExecutorRepository';
```

Comment:
`XRepository` classes are for fetching data. Users can map this to mongo requests etc.